.drone.star

```diff
@@ -341,6 +343,25 @@ def build_client_docs(ctx):
         },
     }
 
+def  gui_tests_format():
+    return {
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "guitestformat",
+        "steps": [
+            {
+                "name": "black",
+                "image": "cytopia/black",
+                "pull": "always",
+                "commands": [
+                    'cd /drone/src/test/gui',
+                    'black --check --diff .',
```

Comment:
It is better to include the steps to check the lining locally in some documentation as well.
Everything else looks good to me 👍🏽